laboratory/src/org/labkey/laboratory/LaboratoryController.java

```diff
@@ -1029,7 +1032,10 @@ public void export(ProcessAssayForm form, HttpServletResponse response, BindExce
             {
                 if (errors.hasErrors())
                 {
-                    HttpView errorView = ExceptionUtil.getErrorView(HttpServletResponse.SC_BAD_REQUEST, "Failed to create template - invalid input", null, getViewContext().getRequest(), false);
+                    ErrorRenderer renderer = ExceptionUtil.getErrorRenderer(HttpServletResponse.SC_BAD_REQUEST, "Failed to create template - invalid input", null, getViewContext().getRequest(), false, false);
```

Comment:
I think we should consider still having the error view be generated and hooked up via `ExceptionUtil`. This is a lot of configuration that is specific to the view that doesn't need to be exposed to callers.

Reply:
Looks good.